src/documents/management/commands/document_exporter.py

```diff
@@ -62,7 +62,9 @@ def dump(self):
             document = document_map[document_dict["pk"]]
 
             target = os.path.join(self.target, document.file_name)
+            thumbnail_target = target + "-tumbnail.png"
             document_dict["__exported_file_name__"] = target
+            document_dict["__exported_thumbnail_name__"] = thumbnail_target
```

Comment:
Now that we've got two arbitrary strings being used to refer to stuff, they really should be re-defined as constants and imported both here and in the importer.  I recommend the following:

1. Create a file at `src/documents/settings.py`
2. Put the two constants in there (Or another reasonable name.  I'm not partial to these.):

```python
EXPORTER_FILE_NAME = "__exported_file_name__"
EXPORTER_THUMBNAIL_NAME = "__exported_thumbnail_name__"
```

3. In the importer and exporter use `from .settings import EXPORTER_FILE_NAME` (or whatever name you like)